src/api/spec/bootstrap/features/webui/maintenance_workflow_spec.rb

```diff
@@ -21,6 +21,7 @@
     create(:maintenance_project_attrib, project: maintenance_project)
   end
 
+  # rubocop:disable RSpec/MultipleExpectations
```

Comment:
Sadly, we need a lot of `expect` as the maintenance workflow is quite exhaustive. We should split this up, but for now, it will have to do.

Reply:
I would disable this cop for feature tests. 15 is a rather random limit

Reply:
I'll submit a PR for this, let's see what the team says.

Reply:
PR: #6555

Reply:
Merged, so removing those comments.